build.gradle

```diff
@@ -21,6 +21,7 @@ dependencies {
     testCompile 'org.powermock:powermock-module-junit4:1.6.2'
     testCompile 'com.sun.jersey:jersey-core:1.19'
     testCompile 'com.netflix.karyon:karyon2-archaius:2.7.1'
+    testCompile group: 'com.tngtech.java', name: 'junit-dataprovider', version: '1.10.0'
```

Comment:
Why don't you use the compact syntax?
```java
testCompile 'com.tngtech.java:junit-dataprovider:1.10.0'
```

It will be more readable if all are in the same format.